Lib/test/test_sqlite3/__main__.py

```diff
@@ -0,0 +1,4 @@
+from test.test_sqlite3 import load_tests
```

Comment:
Why is this necessary if it isn't used?

Reply:
Oh, good question. It's probably a left-over from things I tested earlier today. Sorry 'bout that! :)

Reply:
Aargh, it _is_ used! It's used _implicitly_ by `unittest`. Now `./python.exe -m test.test_sqlite3` fails:

```
HEAD is now at 35f6334d72 Address review: remove useless import
$ ./python.exe -m test.test_sqlite3        
test_sqlite3: testing with version '2.6.0', sqlite_version '3.37.0'

----------------------------------------------------------------------
Ran 0 tests in 0.000s

OK
```